app/frontend/actions/error.ts

```diff
@@ -0,0 +1,34 @@
+import debugLog from '../helpers/debugLog'
+import captureBySentry from '../helpers/captureBySentry'
+import {Store, State} from '../state'
+
+export default ({setState}: Store) => {
+  return {
+    // TODO: remove "options", as "setError" should not be responsible for setting other
+    // parts of the state
+    setError: (
+      state: State,
+      {errorName, error, options}: {errorName: string; error: any; options?: any}
```

Comment:
What is the motivation behind wrapping the "action parameters" in object, e.g. (errorName, error, options) =>  {errorName, error, options}? Is this some pattern we want to follow? Just curious since this makes the usage of the `setError` action a bit more cumbersome in some cases. For example instead of 
```
catch(e) {
  setError(state, 'someError', e) 
}
```
we would need to create the object every time since no current error type has the structure {errorName, error}...

Reply:
The motivation comes just from the fact that in redux it is a common pattern that the reducer receives an object of shape:
```
{type: UNIQUE_ACTION_NAME, payload: {})
```
Where payload is the object containing all params.  I guess that in this example it does not add much value (more verbose), but this was the motivation. What do you think @ppershing / @PeterBenc  should I revert this back?

Reply:
having explicit "options" argument is actually a good pattern in TS as it helps to be more explicit and also provides less potential for wrong argument order.

Reply:
I totally fine with the payload being an object, I was just curious about the motivation. :+1: